tester/runner.go

```diff
@@ -574,3 +574,15 @@ func LoadBundles(args []string, filter loader.Filter) (map[string]*bundle.Bundle
 
 	return bundles, nil
 }
+
+//AggregateBundleModules will retrieve modules from bundles
+func AggregateBundleModules(bundles map[string]*bundle.Bundle, modules map[string]*ast.Module) map[string]*ast.Module {
+	for path, b := range bundles {
+		for bundleName, mod := range b.ParsedModules(path) {
+			splitBundleName := strings.Split(bundleName, "/")
+			name := splitBundleName[len(splitBundleName)-1]
+			modules[name] = mod
+		}
+	}
+	return modules
```

Comment:
🤔 We are we using an argument that's also returned...? I'm kind of confused here. Would this _not_ do the trick?

```suggestion
// AggregateBundleModules will retrieve modules from bundles
func AggregateBundleModules(bundles map[string]*bundle.Bundle) map[string]*ast.Module {
	modules := map[string]*ast.Module{}
	for path, b := range bundles {
		for bundleName, mod := range b.ParsedModules(path) {
			splitBundleName := strings.Split(bundleName, "/")
			name := splitBundleName[len(splitBundleName)-1]
			modules[name] = mod
		}
	}
	return modules
```

Reply:
makes sense ! will change it